internal/interface/jwt.go

```diff
@@ -0,0 +1,67 @@
+package _interface
+
+import (
+	"github.com/dgrijalva/jwt-go"
+
+	"crypto/rsa"
+	"errors"
+	"fmt"
+	"time"
+)
+
+// todo: key rotation
+type Jwt struct {
```

Comment:
Please try to think of some better name for this object. Is it a static entity like JWT or some kind of agent/service?